src/utils/extractReactTokens.ts

```
@@ -0,0 +1,119 @@
+  return cssPrefix.replace(/^pf-v6-/, '').replace(/-+/g, '_')
+}
+
+/**
+ * Extracts all token objects from @patternfly/react-tokens that match a given CSS prefix
+ * 
+ * @param cssPrefix - The CSS prefix (e.g., "pf-v6-c-accordion")
+ * @returns Array of token objects with name, value, and var properties
+ */
+export async function extractReactTokens(
+  cssPrefix: string | string[],
+): Promise<Array<{ name: string; value: string; var: string }>> {
```

ESLint

[error] Array type using 'Array<T>' is forbidden. Use 'T[]' instead.

(`@typescript-eslint/array-type`)

---

[error] Unexpected console statement.

(no-console)